src/components/molecules/InformationRow/InformationRow.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { Text } from '../../atoms/Text/Text';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import './InformationRow.scss';
+
+export const InformationRow = ({ onClick, children, src }) => {
+    return (
+        <button onClick={onClick} className='information__row'>
```

Comment:
I wonder if this have to be a button? I think we should use div instead with onClick. @Veryyapeee correct me if I'm wrong.

Reply:
You are 100% correct